File: Codeforces/384/p1.go
package main

import(
"fmt"
)

func main() {
    var n,a,b int
    fmt.Scanf("%d %d %d",&n,&a,&b)
    var s string
    fmt.Scan(&s)
    eq := s[a-1] == s[b-1]
    if eq {
        fmt.Println(0)
    } else {
        fmt.Println(1)
    }
}

File: Codeforces/406/p1.c
#include<stdio.h>

int main(){
    int a,b,c,d;
    scanf("%d %d %d %d",&a,&b,&c,&d);
    bool f = false;
    for(int i = 0; i < 100 && !f; i++)
        for(int j = 0; j < 100 && !f; j++)
            if(b + i*a == d + j*c)
                printf("%d\n",b+i*a), f = true;
    if(!f)
        printf("-1\n");

    return 0;
}
